lib/ansible/modules/cloud/amazon/ec2_eip_facts.py

```diff
@@ -51,6 +51,24 @@
          - i-987654321
   register: my_vms_eips
 
+# List all EIP addresses using the 'Name' tag as a filter.
+- ec2_eip_facts:
+    filters:
+      tag:Name: www.example.com
```

Comment:
Note to self: this line looks odd, but works when copied, pasted, and tested. 

Thanks @bhandeland for the examples.